src/program/uniforms_storage.rs

```diff
@@ -374,6 +374,16 @@ impl UniformsStorage {
         }
     }
 
+    /// Needs to be called when changing programs, since all prior subroutine uniform state
+    /// is lost and without this call we wouldn't the uniforms.
```

Comment:
this sentence is missing a verb. Also I wonder if this should be exposed in the public API, or should be `pub(crate)`. The point of glium is to not require programmers to think about state.

Reply:
Yeah, makes sense, I made it `pub(crate)` and made the comment a bit clearer (I hope).